Makefile

```diff
@@ -151,7 +151,7 @@ installer-build:
 	VERSION=$(VERSION) \
 	BUILD_TIME=$(shell date +"%d-%m-%yT%T%z") \
 	REVISION=$(COMMIT_SHA) \
-	$(MAKE) release
+	$(MAKE) release-linux
```

Comment:
It will exclude darwin releases from being released via actions, no?
Also, I guess it's better to control this behavior via some environment variables

Reply:
Commited/pushed accidentally

Reply:
Commited/pushed accidentally